bundles/org.openhab.binding.lutron/README.md

```diff
@@ -368,6 +369,33 @@ Thing configuration file example:
 Thing shade libraryshade [ integrationId=33]
 ```
 
+### Blinds [**Beta**]
+
+Each Lutron Sivoia QS Venetian Blind or Horizontal Sheer Blind is controlled by a **blind** thing.
+Besides `integrationId`, it requires that the parameter `type` be set to either "Venetian" for venetian blinds or "Sheer" for horizontal sheer blinds.
+There is no default.
+If discovery is used, the `type` parameter will automatically be correctly set when the **blind** thing is created.
+
+Two channels, *blindliftlevel* and *blindtiltlevel*, with item type Rollershutter and category Rollershutter will be created for each **blind** thing.
+They control the up/down motion and the slat tilt motions of the blinds, respectively.
+Each channel accepts Percent, Up, Down, Stop and Refresh commands.
+Sending a Percent command will cause the blind to immediately move so as to be open the specified percentage.
+You can also read the current setting from each channel.
+It is specified as a percentage, where 0% = closed and 100% = fully open. Movement delays are not currently supported.
+
+**Note:** While a blind is moving to a specific level because of a Percent command, the Lutron system will report the target position for the blind rather than the actual current position.
+While a blind is moving because of an Up or Down command, it will report the previous level until it stops moving.
+
+**Beta Notice:** Support for Sivoia QS blinds is new and has been through very limited testing.
```

Comment:
I'm a bit in doubt about this, our documenation should already be versioned

Reply:
Ok. I removed the "New in 2.5" notes. However, I would like to leave in the "Beta" notice for the new blind thing for now until I get confirmation from other users that position updates are working properly with both types of the Sivoia QS blinds.  It's always a struggle to find people to test new device support for this binding!

Reply:
Please remove everything here about being beta. Beta suggest it's part of some process, which there isn't. If all, you could name it 'experimental'  or mention not well tested. But again how does that look in a few months and when is it well tested? What criteria do you have? I think most new features have not been tested thoroughly, and that ok. If the device itself would be beta than it's logical to add a note. It's good to communicate on the forum if you consider something in beta.